### PR TITLE
Invoke-DbaDbShrink: cast to dbasize

### LINKED ISSUE #7226: [Bug] Invoke-DbaDbShrink - Issue with handling larger db files due to Int32 data type.  Needs to be Int64.
In current form, unable to handle DB files over the capacity of Int32.

### PR DESCRIPTION
**Pull request by lancasteradam**
Invoke-DbaDbShrink: cast to dbasize

## Type of Change

 - [x] Bug fix (non-breaking change, fixes #7226 )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [x] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
 
Minor fix to address the issue mentioned in #7226.

### Approach

Changed from [int] to [dbasize] for the casts done in various places in the command. I wasn't able to test this completely due to the db size requirements, but I did confirm by overriding variables with large numeric values that the errors no longer occurred.

### Commands to test

See the integration tests and the scenario mentioned in #7226.

### CODE REVIEW

File: functions/Invoke-DbaDbShrink.ps1 [hunk 1]
```diff
@@ -222,16 +222,16 @@ function Invoke-DbaDbShrink {
                     $desiredFileSize = $spaceUsed + $desiredSpaceAvailable
 
                     Write-Message -Level Verbose -Message "File: $($file.Name)"
-                    Write-Message -Level Verbose -Message "Initial Size (KB): $([int]$startingSize)"
-                    Write-Message -Level Verbose -Message "Space Used (KB): $([int]$spaceUsed)"
-                    Write-Message -Level Verbose -Message "Initial Freespace (KB): $([int]$spaceAvailable)"
-                    Write-Message -Level Verbose -Message "Target Freespace (KB): $([int]$desiredSpaceAvailable)"
-                    Write-Message -Level Verbose -Message "Target FileSize (KB): $([int]$desiredFileSize)"
+                    Write-Message -Level Verbose -Message "Initial Size (KB): $([dbasize]$startingSize)"
```
Comment (wsmelton) [inline functions/Invoke-DbaDbShrink.ps1:225]: Bit more efficient if we set this when the variables referenced are declared, line 218-222

Comment (lancasteradam) [reply functions/Invoke-DbaDbShrink.ps1:225]: I agree, great catch

**Comment by potatoqualitee** [comment]
Always an honor to get a PR from you, @lancasteradam  🙇🏼  Thanks everyone for the feedback! Merging.

**Comment by lancasteradam** [comment]
Thank you @potatoqualitee for the kind words!

**Pull request merged**